### DISCUSSION TITLE
Acquisition of Apple Enterprise Developers

### DISCUSSION THREAD

**Discussion by zipzhaokai** [CollaboraOnline/online#7536] [Q&A] [answered]
You need an Apple enterprise developer certificate or account. Please contact me with telegram: [removed]

**Comment by Minion3665** [comment, marked as answer]
Hi there! We apologize, but we can't provide you an Apple enterprise developer account. You can buy  one from apple here: https://developer.apple.com/programs/

We're really sorry about this, we understand that the cost of a developer account (and hardware) may be a significant barrier to developing COOL for iOS. If you have any bugs that you notice, patches to test, etc. please feel free to make a bug report here (and ping me if you like!). We do have some macs and developer accounts, and we're happy to test/make fixes for you

**Reply by zipzhaokai** [reply]
I need an Apple Enterprise Developer account.
